api/proto/teleport/trust/v1/trust_service.proto

```diff
@@ -35,3 +37,17 @@ message GetCertAuthorityRequest {
   // Whether the private key should be included in the response.
   bool include_key = 3;
 }
+
+// Request for GetCertAuthorities
+message GetCertAuthoritiesRequest {
+  // Type of certificate authority.
+  string type = 1;
+  // Whether the private key should be included in the response.
+  bool include_key = 3;
+}
+
+// Response for GetCertAuthorities
+message GetCertAuthoritiesResponse {
+  // The matching certificate authorities.
+  repeated types.CertAuthorityV2 cert_authorities = 1;
```

Comment:
Nit: My only suggestion would be to potentially name this field `cert_authorities_v2` or even just `v2`. I've noticed when I was introducing a new version of a resource in Teleport that in a lot of places, the naming has been quite awkward because we've boxxed ourselves into one version of a resource existing.

Reply:
+1, technically even the field names should not be renamed in protobuf (as the field name is used in the protojson encoding).

I'm not sure why we renamed a perfectly serviceable `RoleV3` three times already (up to `RoleV6`) even though versions 3 through 6 are all unmarshaled into the same struct anyway. Maybe we should stop calling protobuf structs like `FooV1` and we should start giving them names like `FooImpl`, `FooImpl2` and so on?